en/includes/references/grant-types.md

```diff
@@ -181,42 +429,125 @@ The following diagram shows how the token exchange grant flow works.
 
 The following diagram illustrates this flow.
 
-![How the organizatoin switch grant works]({{base_path}}/assets/img/references/grants/organization-switch.png)
+![How the organization switch grant works]({{base_path}}/assets/img/references/grants/organization-switch.png)
+
+1. The user visits the client application and attempts to log in.
+2. The client application redirects the authorization code request to the root organization.
+
+    === "Request format (/authorize)"
+
+        ``` bash
+        {{base_url}}/oauth2/authorize
+        ?response_type=code
+        &client_id=<CLIENT_ID>
+        &redirect_uri=<REDIRECT_URI>
+        ```
+
+    === "Sample request (/authorize)"
+
+        ```bash
+        {{base_url_example}}/oauth2/authorize
+        ?response_type=code
+        &client_id=cHuGSLCm77ApRFcsdyh_4sdFU2XYa
+        &redirect_uri=https://localhost:3000
+        ```
 
-1. The user visits the client application and requests login through the organization (root).
-2. The client application redirects the authorization code request to the organization (root).
 3. {{ product_name }} prompts the login page of the root organization's application.
 4. The user selects the **SSO** authentication option.
-5. The organization (root) prompts the user to enter the organization name.
+5. The root organization prompts the user to enter the organization name.
 6. The user enters the organization name.
-7. The organization (root) sends an authorization code request to the organization.
-8. The organization prompts the user to enter credentials.
+7. The root organization sends an authorization code request to the child organization.
+8. The child organization prompts the user to enter credentials.
 9. The user enters the credentials.
-10. The organization sends the authorization code to the organization (root).
-11. The organization (root) uses this authorization code to request an access token from the organization.
-12. The organization sends the access token and ID token to the organization (root).
-13. The organization (root) sends the authorization code to the client application.
-
-    !!! note
-        This is the response to the authorization code request in step two.
-
-14. The client application uses this authorization code to request an access token from the organization (root).
-15. The organization (root) sends the access token and ID token to the client application.
-16. The client application exchanges the access token received in the above step for an access token for the organization.
-17. The organization (root) initiates an exchange for an access token and sends an access token against the organization to the client application.
+10. The child organization sends the authorization code to the root organization.
```

Comment:
Seems the steps and the flow in the digram is incorrect for organization swicth.

1. Invoke /oauth2/authorize request to root org
2. User in the root org authenticate 
3. Application receives the authz code
4. Application exchange the authz code to access token
5. Now user has an access token to access resources in root
6. if the same user has access granted in a sub organization (user has a shared user reference through sub org invite/ shared from org creation) he can switch the first access token to sub org token using organization switch grant